### DISCUSSION TITLE
TJvWizardPageHeader does not handle High-DPI scaling properly

### DISCUSSION THREAD

**Discussion by adrienreboisson** [project-jedi/jvcl#177] [Q&A]
Hi,

First sorry for posting here but I don't understand how to create an account on your mantis bugtracker. 

I'm slowly starting to add high-DPI support for my applications. I noticed there is a bug for the `TJvWizardPageHeader `component, which is not scaled properly on runtime when `dpiAware` is set to `true` in the app manifest. 

Inside the constructor, its height is set manually to a fixed amount of pixels :

```
constructor TJvWizardPageHeader.Create;
begin
  inherited Create;
  ...
  FHeight := 70; //<----
```

I think  `PPIScale()` should be used here to fix the issue.

Best regards,

Adrien

**Comment by obones** [comment]
Can you confirm that using `PPIScale` works?

**Comment by adrienreboisson** [comment]
Yes, I just checked on my computer and it works.

Should I do a pull request ?
